scripts/publish.sh

```diff
@@ -30,23 +30,23 @@ npm install
 
 # Node client
 bump_and_npm_publish "@nangohq/node" "$VERSION"
-pushd "./packages/shared"; npm install "@nangohq/node@$VERSION"; popd;
+pushd "$GIT_ROOT_DIR/packages/shared"; npm install -E @nangohq/node@^$VERSION; popd;
```

Comment:
are you sure? you have -E and the caret :D

Reply:
oups. forgot to remove -E